sdk/endpoint/switch_builder.go

```diff
@@ -0,0 +1,88 @@
+package endpoint
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/networkservice"
+)
+
+// SwitchEndpointBuilder is a helper to construct SwitchEndpoint instance
+// which matches incoming requests based on Connection.NetworkService and
+// Connection.Labels.
+type SwitchEndpointBuilder struct {
```

Comment:
I do not think that actually we need this entity because it could look complex for the user